### PR TITLE
Added a "Do" for Hosting Discussion

### PR DESCRIPTION
**Pull request by connor4312**
See the issue: https://github.com/minecraftforum/rules/issues/183

### CODE REVIEW

File: rules/servers/rules.markdown [hunk 1]
```diff
@@ -91,6 +91,15 @@ Servers is the section for PC specific posts to be made.
 
 #### Hosting Discussion
 
+__Do__
+
+Following is a list of actions which are encouraged, to give an idea what the Hosting Discussion section is for. You are *not* limited to only what is included below.
+
+ * Requests staff or aid for your hosting work.
```
Comment (LordRalex) [inline rules/servers/rules.markdown]: That seems grammatically incorrect

File: rules/servers/rules.markdown [hunk 2]
```diff
@@ -91,6 +91,15 @@ Servers is the section for PC specific posts to be made.
...
+ * Post guides relating to the hosting of servers.
+ * Share websites relating to or assisting with server hosting.
```
Comment (LordRalex) [inline rules/servers/rules.markdown]: This would seem to counter the advertising rule, is also not really clear on what exactly would be allowed.

**Comment by LordRalex** [comment]
Please format your PR to follow the styles: https://github.com/minecraftforum/rules/blob/master/README.markdown#style--format-guide

Specifically the "you" bits and the full stops

**Comment by connor4312** [comment]
Fixed. I also removed the full stops from the "Do Not" section below, to fit the formatting guide.

**Comment by LordRalex** [comment]
Okay, looks much better. We'll confirm it is good, then pull it when we can

**Pull request merged**